contk/dataloader/language_generation.py

```diff
@@ -111,6 +111,23 @@ def get_inference_metric(self, gen_key="gen"):
 													 gen_key=gen_key))
 		return metric
 
+	def get_gan_metric(self, gen_key="gen_gan"):
```

Comment:
It seems that self-bleu and forward/backward bleu should be added in get_inference_metric. Refer to contk/dataloader/single_turn_dialog.py